app/taskqueue/tasks/kcidb.py

```diff
@@ -40,10 +43,15 @@ def push_build(args):
 @taskc.app.task(name="kcidb-tests")
 def push_tests(group_id):
     kcidb_options = taskc.app.conf.get("kcidb_options")
+    pid = os.getpid()
+    submit_process = taskc.app.kcidb_pool.get(pid)
```

Comment:
Either use `[pid]` if it's guaranteed to be there, or check if `submit_process` is `None` before using it.

I guess the expectation is that `[pid]` should always work, as this shouldn't get called unless the hooks have been invoked when starting a worker process.

Reply:
I changed it to `[pid]`, it was a leftover from development phase of this fix.